src/app/client/src/app/modules/resource/components/collection-player/collection-player.component.ts

```diff
@@ -225,7 +229,7 @@ export class CollectionPlayerComponent implements OnInit, OnDestroy {
       object: {
         id: this.collectionId,
         type: this.collectionData.contentType,
-        ver: this.collectionData.pkgVersion ? this.collectionData.pkgVersion.toString() : '1.0'
+        ver: this.collectionData.pkgVersion || '1'
```

Comment:
the same change is repeated in multiple places in this file. any specific reason for doing this?